File: anagram/anagram.cpp
#include <stdio.h>
#include <iostream>
#include <string>
#include <algorithm>
using  namespace std;
int main()
{
	string str1, str2;

	printf("Enter the first string : ");
	cin >> str1;

	printf("Enter the second string : ");
	cin >> str2;

	sort(str1.begin(), str1.end());
	sort(str2.begin(), str2.end());
	cout << str1 << endl;
	cout << str2 << endl;

	if( str1 == str2 )
		printf("Anagram \n");
	else
		printf("Not An Anagram \n");
}

File: anagram/anagram.py

str1 = input("Enter the first string : ");
str2 = input("Enter the second string : ");

freq = [0] * 26

for c in str1:
	freq[ ord(c) - 65 ] += 1
print(freq)

for c in str2:
	freq[ ord(c) - 65 ] -= 1
print(freq)
print("Anagram" if any(freq) == False else "Not An Anagram")
